pkgs/kcl-lang/kcl/registry.yaml

```diff
@@ -0,0 +1,18 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/registry.json
+packages:
+  - type: github_release
+    repo_owner: kcl-lang
+    repo_name: kcl
+    description: KCL Programming Language (CNCF Sandbox Project). https://kcl-lang.io
+    asset: kclvm-{{.Version}}-{{.OS}}-{{.Arch}}.{{.Format}}
+    format: tar.gz
+    overrides:
+      - goos: windows
+        asset: kclvm-{{.Version}}-{{.OS}}.{{.Format}}
+        format: zip
+        files:
+          - name: kclvm_cli.exe
+            src: bin/kclvm_cli.exe
+    files:
+      - name: kclvm_cli
```

Comment:
```suggestion
      - name: kcl
```

I think the command name should be kcl.

https://www.kcl-lang.io/docs/user_docs/getting-started/install/#note

Reply:
Oh, I see. This tool includes a lot of commands.
And kclvm_cli is different from kcl.

```
kclvm/bin/kcl
kclvm/bin/kcl-doc
kclvm/bin/kcl-fmt
kclvm/bin/kcl-language-server
kclvm/bin/kcl-lint
kclvm/bin/kcl-plugin
kclvm/bin/kcl-test
kclvm/bin/kcl-vet
kclvm/bin/kclvm
kclvm/bin/kclvm_cli
```

Reply:
I'm supposed to use https://github.com/kcl-lang/cli instead 🤦 I'll update the PR tomorrow. Apologies for the confusion!

Reply:
Oh, I see.
Could you create a new PR for https://github.com/kcl-lang/cli ?
I'm fixing this pr.
I'm not familiar with these tools, but maybe we can support them too.